asa_cli/api.py

```
@@ -205,6 +206,28 @@ def org_id(self) -> int:
+            response = self._request("GET", "/campaigns", params={"limit": 1, "offset": 0})
+            campaigns = response.get("data", []) if isinstance(response, dict) else []
+            if campaigns:
+                campaign = campaigns[0]
+                daily_budget = campaign.get("dailyBudgetAmount", {})
+                currency = daily_budget.get("currency")
+                if currency:
+                    self._currency = str(currency)
+                    return self._currency
+        except Exception:
+            # Best-effort only; fall back to USD below
+            pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)